techniques/system/common/1.0/restart-services.st

```diff
@@ -0,0 +1,63 @@
+#####################################################################################
+# Copyright 2015- Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+#######################################################
+#
+# restart_services
+#
+# This bundle restarts all the services in need at the end
+# of the agent execution
+#
+#######################################################
+
+bundle agent restart_services
+{
+  
+  methods:
+  
+    # (sys)log service
+
+    (!SuSE.!aix.!solaris).syslogd_repaired::
+
+      "restart_syslog" usebundle => service_restart("syslog");
+
+    syslog_ng_repaired.!SuSE::
+
+      "restart_syslog_ng" usebundle => service_restart("syslog-ng");
+
+    (syslog_ng_repaired|rsyslog_repaired|syslogd_repaired).!SuSE::
```

Comment:
this one was there before, but looks absurd.
it should only be
syslogd_repaired::
 "restart_syslog" usebundle => service_restart("syslog");